From 29f1a37bbd171c4e6692ad6851d505cb4ddc60c8 Mon Sep 17 00:00:00 2001
From: Haibo Chen <haibo.chen@freescale.com>
Date: Thu, 26 Mar 2015 13:52:37 +0800
Subject: [PATCH] MLK-10468 ARM: dts: imx6qp-sabreauto.dts: turn on vgen3_reg
 regulator

Make the vgen3_reg regulator always on for i.MX6QP-sabreauto board.
This vgen3_reg regulator supply 1.8v for SD3.0 mode.

Signed-off-by: Haibo Chen <haibo.chen@freescale.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto.dts b/arch/arm/boot/dts/imx6qp-sabreauto.dts
index bddfe12..9aa2b9b 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto.dts
@@ -25,3 +25,7 @@
 		#gpio-cells = <2>;
 	};
 };
+
+&vgen3_reg {
+	regulator-always-on;
+};
-- 
1.8.0

